CHANGELOG-3.1.md

```diff
@@ -0,0 +1,12 @@
+CHANGELOG for 3.1.x
+===================
+
+This changelog references the relevant changes (bug and security fixes) done
+in 3.1 minor versions.
+
+To get the diff for a specific change, go to https://github.com/symfony/symfony/commit/XXX where XXX is the change hash
+To get the diff between two versions, go to https://github.com/symfony/symfony/compare/v3.1.0...v3.1.1
+
+* 3.1.0
+
+ * bug #17611 [HttpKernel] Deprecate passing objects as URI attributes to the ESI and SSI renderers
```

Comment:
This file is auto-generated. I meant the CHANGELOG file for the HttpKernel Component instead. Sorry for the confusion.